packages/swig/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=http://swig.org
+TERMUX_PKG_DESCRIPTION="Generate scripting interfaces to C/C++ code"
+TERMUX_PKG_VERSION="3.0.12"
+TERMUX_PKG_BUILD_IN_SRC=yes
+TERMUX_PKG_DEPENDS="boost, pcre, pcre-dev, perl"
```

Comment:
1. Boost is not needed to build swig.
2. Remove pcre-dev - just pcre is enough.